Sources/System/FilePath/FilePathComponents.swift

```diff
@@ -190,6 +190,7 @@ extension FilePath.Root: _PathSlice {
   }
 }
 extension FilePath: _PlatformStringable {
+  @usableFromInline
```

Comment:
@lorentey we will need to version this for `Darwin` binary distribution, but otherwise I think this is an ok workaround.

Reply:
Adding `@usableFromInline` would introduce this as ABI, which we don't actually need/want to do on Darwin. (Its availability wouldn't match the rest of FilePath, making it inconvenient.)

I'll try to figure out an alternative solution. I'm not sure why the caller needs to be marked `@_alwaysEmitIntoClient` on Windows, for example.